engine/runtime/src/main/java/org/enso/ClassLoaderConstants.java

```diff
@@ -13,9 +13,9 @@ public class ClassLoaderConstants {
    * consistent.
    */
   public static final List<String> CLASS_DELEGATION_PATTERNS =
-      List.of("org.graalvm", "java", "org.slf4j", "ch.qos");
+      List.of("org.graalvm", "java", "org.slf4j", "ch.qos", "org.jline");
```

Comment:
We could potentially not have to delegate to system class loader, if we would make sure that no `org.jline` packages are in `runner.jar` fat jar. Let's add it to the delegation pattern just to be sure.

Reply:
You can call me heretic, but:

- `runtime.jar` shall not use `jline` at all - it is an implementation of a `TruffleLanguage`, not a launcher
- thus `runner.jar` shall be the only one using `jline`

If that's the case, I'd prefer to put jline into runner's fat JAR. Don't get me wrong, I am no fan of fat JARs, but I'd like to keep the `--module-path` only for Truffle for now....

Reply:
`jline` is already inside `runner.jar` fat jar. The `EngineRunnerBootLoader` successfully finds the `org.jline` classes inside `runner.jar`, but the initialization of the terminal fails, and a dumb terminal is provided. I was trying to provide `org.jline` only inside `runner.jar` but failed to do so. Let's just add it on the `--module-path`. We have logging utilities there anyway.

Reply:
Logging is used by both systems. 

>  the initialization of the terminal fails, and a dumb terminal is provided

Why it fails? Wrong `ClassLoader` used to read resources or load classes?

Reply:
My goal is to run `runner.jar` into a library inside of `Standard.Visualization` module. By moving parts of `runner.jar` into `--module-path` you are not making my goal easier.

Reply:
It seems to me that https://github.com/enso-org/enso/issues/7082 is a blocker for entering `runner.jar` from `Standard.Visualization` library, right? Can we theoretically solve https://github.com/enso-org/enso/issues/7082 by introducing a new `ModuleLayer` for each library (more specifically, only for libraries that have `jar` archives in their `polyglot` directory), similarly to how Truffle handles class path isolation [here](https://github.com/oracle/graal/blob/0b14c1e31ba02def0b6d28fa230c3d4f30e27dab/sdk/src/org.graalvm.polyglot/src/org/graalvm/polyglot/Engine.java#L1760)? If the answer is yes, then there is no problem in introducing yet another module.